hadoop-tools/hadoop-aws/pom.xml

```diff
@@ -483,7 +483,22 @@
     </dependency>
     <dependency>
       <groupId>com.amazonaws</groupId>
-      <artifactId>aws-java-sdk-bundle</artifactId>
+      <artifactId>aws-java-sdk-core</artifactId>
```

Comment:
Does this dep need to be explicit? You should get this transitively.

Reply:
Let me check that. I explicitly added this later, but I can't recall which class it supplied that was missing.

Reply:
You were right. I've removed the core dependency.